src/gui/folderwizard.cpp

```diff
@@ -367,7 +367,7 @@ void FolderWizardRemotePath::slotUpdateDirectories(const QStringList &list)
     }
     QStringList sortedList = list;
     Utility::sortFilenames(sortedList);
-    foreach (QString path, sortedList) {
+    for (QString path : sortedList) {
```

Comment:
better to take by reference to avoid creating a copy

```suggestion
    for (const auto &path : sortedList) {
```

Reply:
Hi! Thanks for the feedback! In this case, shouldn´t it be without const, as the variable is modified within the loop?

Reply:
Oops, sorry I missed that -- in this case, just `auto path` is fine :)